app/Http/Controllers/DashboardV2Controller.php

```
@@ -45,31 +51,127 @@ public function __construct()
+     *
+     * Every panel is deferred and grouped so the shell paints immediately: the
+     * queue and its board columns land together (a row cannot render its status
+     * control without them), the rest arrive independently.
+     */
+    private function personalDashboard(
+        $user,
+        DashboardMetricsService $metrics,
+        array $availableViews
+    ) {
+        $userId = (int) $user->id;
+        $companyId = (int) $user->company_id;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$companyId'. (undefined)

(UnusedLocalVariable)